nx.json

```diff
@@ -10,7 +10,18 @@
   },
   "targetDefaults": {
     "build": {
-      "inputs": ["{workspaceRoot}/scripts/*", "{workspaceRoot}/lerna.json"]
```

Comment:
This will enforce running build and test when the workflows are updated. Otherwise, something might be skipped if there are no changes on the actual packages